subsys/net/lib/zperf/zperf_udp_uploader.c

```diff
@@ -246,15 +245,19 @@ void zperf_udp_upload(const struct shell *sh,
 			nb_packets++;
 		}
 
+#ifdef CONFIG_NET_ZPERF_LOG_LEVEL_DBG
```

Comment:
This could be a C `if (IS_ENABLED(CONFIG_NET_ZPERF_LOG_LEVEL_DBG)) {` just to make sure the code still compiles in the future.